accesscontrol_usermanagement/configuring-argo-cd-rbac.adoc

```diff
@@ -4,9 +4,14 @@ include::_attributes/common-attributes.adoc[]
 = Configuring Argo CD RBAC
 :context: configuring-argo-cd-rbac
 
-toc::[]
+By default, any type of user except the `kube:admin` user, logged into Argo CD does not have access to any services by default.
```

Comment:
```suggestion
By default, any type of user except the `kube:admin` user, logged into default `openshift-gitops` Argo CD instance does not have access to any services by default.
```

Reply:
Hi @svghadi - I would refrain from using `openshift-gitops` explicitly as the user is aware of the disitinction between a default and a custom Argo CD instance.

Reply:
I have included the remaining part of your feedback in the content edit.